jsettlers.main.android/src/main/java/jsettlers/main/android/Menus/GameMenu.java

```diff
@@ -0,0 +1,194 @@
+package jsettlers.main.android.menus;
+
+import android.app.Notification;
+import android.app.NotificationManager;
+import android.app.PendingIntent;
+import android.content.Context;
+import android.content.Intent;
+import android.support.v4.app.NotificationCompat;
+import android.support.v4.content.LocalBroadcastManager;
+
+import java.util.Timer;
+import java.util.TimerTask;
+
+import go.graphics.android.AndroidSoundPlayer;
+import jsettlers.common.menu.action.EActionType;
+import jsettlers.graphics.action.Action;
+import jsettlers.graphics.map.MapContent;
+import jsettlers.main.android.R;
+import jsettlers.main.android.activities.GameActivity;
+import jsettlers.main.android.navigation.Actions;
+
+import static jsettlers.main.android.GameService.ACTION_PAUSE;
+import static jsettlers.main.android.GameService.ACTION_QUIT;
+import static jsettlers.main.android.GameService.ACTION_QUIT_CANCELLED;
+import static jsettlers.main.android.GameService.ACTION_QUIT_CONFIRM;
+import static jsettlers.main.android.GameService.ACTION_SAVE;
+import static jsettlers.main.android.GameService.ACTION_UNPAUSE;
+
+/**
+ * Created by tompr on 19/11/2016.
+ */
+
+public class GameMenu {
+    public static final int NOTIFICATION_ID = 100;
+
+    private Context context;
+    private LocalBroadcastManager localBroadcastManager;
+    private NotificationManager notificationManager;
+
+    private MapContent mapContent;
+    private AndroidSoundPlayer soundPlayer;
+    private Timer quitConfirmTimer;
+
+    private boolean paused = false;
+
+    public GameMenu(Context context, MapContent mapContent, AndroidSoundPlayer soundPlayer) {
+        this.context = context;
+        this.mapContent = mapContent;
+        this.soundPlayer = soundPlayer;
+
+        localBroadcastManager = LocalBroadcastManager.getInstance(context);
+        notificationManager = (NotificationManager) context.getSystemService(Context.NOTIFICATION_SERVICE);
+    }
+
+    public void save() {
+
+    }
+
+    // mute the game when pausing whether or not its currently visibile
+    public void pause() {
+        mapContent.fireAction(new Action(EActionType.SPEED_SET_PAUSE));
+        mute();
+        paused = true;
+
+        // Send a local broadcast so that any UI can update if necessary
```

Comment:
Plase use the game events for this - in multiplayer, the game could be paused by any player. And since game state is updated asynchronously, the pause command will be executed a few (milli)seconds after this method is called.